crates/nu-command/src/strings/split/list.rs

```diff
@@ -95,6 +95,24 @@ impl Command for SubCommand {
                     span: Span::test_data(),
                 }),
             },
+            Example {
+                description: "Split a list of chars into two lists",
+                example: "[a, b, c, d, e, f, g] | split list a",
```

Comment:
Would you mind changing this to doing some splitting other than just removing the first item in the list? Maybe something like `[a, b, c, d, a, e, f, g] | split list a`? At least then we end up with two lists, which I think is the intent. The way this example is now could be confusing.

Reply:
Ok, that's a good point :)